osu.Game/Screens/Select/SongSelect.cs

```diff
@@ -225,10 +225,12 @@ private void load(BeatmapManager beatmaps, AudioManager audio, DialogOverlay dia
 
         public void Edit(BeatmapInfo beatmap)
```

Comment:
If the only use of this method is through `EditSelected()`, would rather just inline `beatmapNoDebounce` in here and invoke `Edit()` from places where `EditSelected()` is currently invoked.